### ISSUE TITLE
navigate.nextDoc with Ctrl-PageDown

### ISSUE THREAD

**Issue by pavlovt** [adobe/brackets#10233]
Hi,
I would like to navigate to prev /next open document with Ctrl+PageUp / Ctrl+PageDown
but when entered in brackets.json (Brackets 1.1)
```json
"navigate.nextDoc": {
            "commandID": "navigate.nextDoc",
            "key": "Ctrl-PageDown"
        }
```
it is not working.
Please tell me if there is a way to do it.

**Comment by redmunds** [comment]
Confirmed. Closing.

@pavlovt This fix will be in Brackets 1.2

**Comment by pavlovt** [comment]
Thanks :)
